source/configuration/write.txt

```diff
@@ -114,6 +124,15 @@ The following options for writing to MongoDB are available:
          guide on the
          :manual:`WriteConcern j option </reference/write-concern/#j-option>`.
 
+   * - ``writeConcern.w``
+     - | Specifies ``w``, a write-concern option to request acknowledgment
+         that the write operation has propogated to a specified number of
+         MongoDB instances. For a list
```

Comment:
Suggestion:
Use "nodes" instead of "instances" when referencing a replica set to make it clear that they are part of the same replica set. Example usage: e.g.: https://www.mongodb.com/basics/clusters